bin/kytos

```diff
@@ -26,26 +26,42 @@ The most commonly used kytos commands are:
 See 'kytos <command> -h|--help' for more information on a specific command.
 """
 import logging
+import os
+
 from docopt import docopt
+from kytos.utils.config import KytosConfig
 
 logging.basicConfig(format='%(levelname)-5s %(message)s', level=logging.INFO)
 
 if __name__ == '__main__':
+    kytos_config = KytosConfig()
+    kytos_metadata = kytos_config.get_remote_metadata()
+    kutils_metadata = kytos_config.get_metadata()
+    kytos_version = kytos_metadata.get('__version__')
+    kutils_version = kutils_metadata.get('__version__')
+
     args = docopt(__doc__,
-                  version='kytos command line, version 2019.2b2',
+                  version='kytos command line, version %s' % kutils_version,
                   options_first=True)
     command = args['<command>']
     command_args = args['<args>']
     argv = [command] + command_args
 
+    if kytos_version != kutils_version:
+        logger = logging.getLogger()
+        logger.warning('kytos (%s) and kytos utils (%s) versions '
+                       'are not equal.', kytos_version, kutils_version)
```

Comment:
This should move to another module, preferably be checked every time someone tries to connect to a remote kytos daemon.